.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+Please help potential reviewers to understand this pull request and speed up 
+the process by answering the following questions, where appropriate.
+
+# Why was this pull request created?
+
+# How is this achieved?
+
+# How to test this?
```

Comment:
Firstly this is not about per-commit validation, since it's a description of how to test the PR as a whole.

Secondly, it doesn't need to be per-PR validation. Even if maintenance QA aren't validating *individual* PRs, surely they are validating batches of changes which include bugfixes from PRs, therefore they still need to know how to validate each fix?  Or am I misunderstanding something, and QA actually don't test any of our bugfixes?